### PR TITLE
fix: disable requestIdleCallback for CLS and INP reports

### PR DESCRIPTION
**Pull request by SuperOleg39**

### CODE REVIEW

File: src/reportPerf.ts [hunk 1]
```diff
@@ -31,5 +31,13 @@ export const reportPerf = (
       rating,
       navigationType,
     });
-  });
+  };
+
+  // Send CLS and INP metrics immediately,
+  // because this metrics are reported when page is hidden or closed
+  if (['CLS', 'INP'].includes(measureName)) {
```
Comment (Zizzamia) [inline src/reportPerf.ts:38]: @SuperOleg39 would you mind add tests for this fix please.

Comment (SuperOleg39) [reply src/reportPerf.ts:38]: Done!

**Comment by robertwt7** [comment]
nice job, can't wait for this to be merged!

**Review by Zizzamia — APPROVED** [review]

**Comment by Zizzamia** [comment]
Thank you for adding the test.
I wonder if this will be enough to make the report working.

**Pull request merged**